README.md

```diff
@@ -13,36 +13,37 @@ According to [yyoncho](https://www.reddit.com/r/emacs/comments/ymrkyn/comment/iv
 
 ## How this project works
 
 This project provides a wrapper-executable around lsp server programs, to work around the above-mentioned issues:
 
-- It converts json responses from the server into **elisp bytecode** (in text representation) for Emacs to read.
+- It converts json messages from the server at high speed directly into **elisp bytecode** (in text representation) for Emacs to read.
     * e.g. `{"objs":[{"a":1},{"a":2}]}` would be converted to `#[0 "\301\302\300\303D\300\304D\"D\207" [:a :objs vector 1 2] 13]`
-    * This improves the message parsing performance in Emacs by ~4x for large json objects, see benchmark result [here](https://github.com/blahgeek/emacs-lsp-booster/actions/runs/7416840025/job/20182439682#step:5:142)
+    * This improves the message parsing performance in Emacs by ~4x for large json objects; see benchmark result [here](https://github.com/blahgeek/emacs-lsp-booster/actions/runs/7416840025/job/20182439682#step:5:142)
     * Although Emacs still needs to parse the text representation and interpret it into elisp objects, the performance gain mainly comes from the following:
         * Parsing (`read`ing) elisp object is apparently better optimized and simpler in Emacs
-        * Using bytecode to construct objects, we can eliminate duplicated objects (e.g. the "a" json key in above example)
-- It separates reading and writing into different threads and keeps pending messages in internal buffers, to avoid blocking on IO, which solves the above-mentioned issues (2) and (3).
+        * By using bytecode to construct objects, we can eliminate duplicated objects (e.g. the "a" json key in above example)
+- It separates reading and writing into different threads and keeps pending messages in internal buffers within each thread, to avoid blocking on IO.  This solves issues (2) and (3) mentioned above.
 
-Overall, this achieves similar result as the native async non-blocking jsonrpc approach without requiring modifications in Emacs source code.
+Overall, this _lsp server wrapper_ strategy achieves similar result as the native async non-blocking jsonrpc approach without requiring modifications in Emacs source code.
 
+> [!IMPORTANT]  
+> At present only local lsp server programs which communicate by standard input/output can be wrapped, not servers communicating over network ports (local or remote).
 
 ## How to use
 
 Generally, what you need to do is:
 
 1. Wrap your lsp server command with this `emacs-lsp-booster` executable.
-   For example, if the original lsp server command is `pyright-langserver --stdio`, configure lsp-mode or eglot to run `emacs-lsp-booster pyright-langserver --stdio` instead.
-2. Advise/update the json parsing function in lsp-mode or eglot to try to parse the input as bytecode prior to parsing it as json.
+   For example, if the original lsp server command is `pyright-langserver --stdio`, configure lsp-mode or eglot to run `emacs-lsp-booster [flags] pyright-langserver --stdio` instead.
```

Comment:
Actually the syntax is either `emacs-lsp-booster [OPTIONS] -- <SERVER_CMD>...` (Note the `--` separator), or `emacs-lsp-booster <SERVER_CMD>...` (no options, for backward compatibility).

Reply:
However you prefer!